src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -886,10 +981,15 @@ internal void ResolveConflict([NotNull]string docID)
             });
         }
 
+        internal long PurgeExpiredDocs(C4Database* c4db, C4Error* outError)
+        {
+            return Native.c4db_purgeExpiredDocs(_c4db, outError);
```

Comment:
Don't call these directly, use `LiteCoreBridge.Check` to catch error conditions

Reply:
PurgeExpiredDocs() is not in the API.